compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/RegExpLexer.java

```diff
@@ -364,9 +364,20 @@ private void processAbbrWithOther() {
             case 'o':
             case 'n':
                 this.reader.advance();
-                break;
+                return;
             default:
-                break;
+                checkInvalidTokenInUnicodeGeneralCategory();
+        }
+    }
+
+    private void checkInvalidTokenInUnicodeGeneralCategory() {
```

Comment:
I think we dont need this change. we can simply move the default block in the switch of the `processReUnicodeGeneralCategoryAbbr()` method out and place it after the switch to get the things done

Reply:
Will check

Reply:
Updated, Please dcheck